src/screens/DiscoveryScreenV2/fragment/DomainFragment.js

```diff
@@ -243,6 +258,9 @@ const DomainFragment = ({
   };
 
   const renderItem = ({from, item, index, section}) => {
+    const isFollow = JSON.stringify(ifollow.map((i) => i.domain_id_followed)).includes(
+      item.domain_id_followed
+    );
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The mechanism for verifying whether a domain is being followed has been enhanced. This is a good change as it improves the accuracy of the follow status.
```diff
+ const isFollow = JSON.stringify(ifollow.map((i) => i.domain_id_followed)).includes(
+   item.domain_id_followed
+ );
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->